### PR TITLE
impr: Add possibility of stubbing the organisations query

### PR DESCRIPTION
**Pull request by akomisarek**
### Change description ###

* wiremock docker image
* currently just returns the same response no matter what

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: docker/wiremock/mappings/organisations.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "request": {
+    "method": "GET",
+    "urlPath": "/refdata/external/v1/organisations/users",
+    "queryParameters": {
+      "status": {
+        "equalTo": "ACTIVE"
+      }
+    }
+
```
Comment (geek-soft) [inline docker/wiremock/mappings/organisations.json]: Unnecessary blank line.

File: docker/wiremock/mappings/index.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
```
Comment (geek-soft) [inline docker/wiremock/mappings/index.json]: Is that needed?

File: docker/wiremock/__files/swanseaOrg.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "userIdentifier": "acfe91d3-745f-4e60-8272-2cada36152a5",
+    "firstName": "damian@swansea.gov.uk",
+    "lastName": "(local-authority)",
+    "email": "damian@swansea.gov.uk",
+    "roles": [
+      "caseworker",
+      "caseworker-publiclaw",
+      "caseworker-publiclaw-solicitor"
+    ],
+    "idamStatus": "ACTIVE?",
```
Comment (geek-soft) [inline docker/wiremock/__files/swanseaOrg.json]: Out of curiosity what does these questions marks mean?

Comment (akomisarek) [reply docker/wiremock/__files/swanseaOrg.json]: That I was not sure at the time of writing what the endpoint returns (as the API is string and not enum...)

File: docker/wiremock/mappings/index.json (hunk 2 above)
Comment (akomisarek) [reply docker/wiremock/mappings/index.json]: No happy to remove, I wanted to show different way of stubbing it's not required.

File: docker/wiremock/mappings/organisations.json (hunk 1 above)
Comment (akomisarek) [reply docker/wiremock/mappings/organisations.json]: Done.

File: docker/wiremock/__files/swanseaOrg.json (hunk 3 above)
Comment (akomisarek) [reply docker/wiremock/__files/swanseaOrg.json]: Removed the question marks.

**Pull request merged**